test/unit/kubernetes-deploy/runner_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class RunnerTest < Minitest::Test
+  def setup
+    @logger_stream = StringIO.new
+    @logger = Logger.new(@logger_stream)
+    KubernetesDeploy.logger = @logger
+  end
+
+  def teardown
+    @logger_stream.close
+  end
+
+  def test_that_it_has_a_version_number
+    refute_nil ::KubernetesDeploy::VERSION
+  end
+
+  def test_initializer
+    # TODO: Switch to --kubeconfig for kubectl shell out and pass env var as arg to Runner init
+    # Then fix this crappy env manipulation
+    original_env = ENV["KUBECONFIG"]
+    ENV["KUBECONFIG"] = "/this-really-should/not-exist"
```

Comment:
To explain my comment, I'm planning to add a `--kubeconfig` option to the CLI, which will default to `ENV["KUBECONFIG"]`, such that the runner will always receive a path as an argument. It'll then use that path in `kubectl [things] --kubeconfig=/path/to/config` instead of relying on the env var.  Then this test can be run without manipulating the environment. I'm in favour of wrappers like that in general, but in this case it feels like putting 💄  on something shitty that's used in a single test. If a second test needs to manipulate this even temporarily, I'll add a helper.